### PR TITLE
Story ST-40

### PR DESCRIPTION
**Pull request by flyingfrog81**
I'd like to collect feedback from all SMs about the description I drafted as part of the documentation. Please report changes as part of the review of this pull-request when reviewing the content of teams_responsibilities.rst file.

### CODE REVIEW

File: src/community/teams_responsibilities.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+SKA software development is organized with agile development teams.
+
+Development team
+================
+
+
+Scrum Master
+============
+The Scrum Master of each team is the responsible for the process the team
```
Comment (rbrederode) [inline src/community/teams_responsibilities.rst]: Fix grammar...."_is the responsible_" should be "_is responsible_".

File: src/community/teams_responsibilities.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+follows. A generic description of this role can be found `on the SAFe website <https://www.scaledagileframework.com/scrum-master/>`_.
+The SKA Scrum Masters are responsible for:
+
+  *  Meet the team, make sure they know each other and find a nice way to present interests, skills and get to know each other. Lead the team to find a name they like.
+
+  * Make sure all team components can access SKA confluence and jira.
```
Comment (rbrederode) [inline src/community/teams_responsibilities.rst]: team members...not team components
Should probably add video conferencing facilities to this list.

File: src/community/teams_responsibilities.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Development team
+================
+
+
+Scrum Master
+============
+The Scrum Master of each team is the responsible for the process the team
+follows. A generic description of this role can be found `on the SAFe website <https://www.scaledagileframework.com/scrum-master/>`_.
+The SKA Scrum Masters are responsible for:
```
Comment (rbrederode) [inline src/community/teams_responsibilities.rst]: The list that follows is not a complete list of responsibilities...e.g. removing impediments, managing risks, protecting the team. Some of these things are described in the SAFe website. Maybe this sentence should start with "The SKA Scrum Masters are also responsible for:" to indicate that these are additional responsibilities to the generic description.

**Review by rbrederode — CHANGES_REQUESTED** [review]

**Comment by jbmorgado** [comment]
So far, I have felt the need for some sort of calendaring tool stating the availability of each team member.

Maybe we could add that to the SM responsibilities and suggest the tool to do it.

**Review by rbrederode — APPROVED** [review]

**Pull request merged**